roles/bootserver/tasks/main.yml

```diff
@@ -63,6 +63,18 @@
   notify:
     - 'bootserver : Restart dnsmasq'
 
+- name: Bootloader symlinks
+  ansible.builtin.file:
+    src: "{{ item.src }}"
+    dest: "{{ item.dest }}"
+    owner: dnsmasq
+    group: tftp
```

Comment:
```suggestion
    owner: root
    group: root
```

This is causing the destination files to also be owned by dnsmasq:tftp, which prevents the node from booting.

First error below is before any changes.
Second error is from `chown root:root` the `/usr/share/tftpboot-installation/SLE-15-SP5-x86_64/EFI/BOOT/bootx64.efi` file.
The third try (not pictured) worked after `chown root:root` on `/usr/share/grub2/x86_64-efi/grub.efi`.
![Screenshot 2023-09-08 at 12 20 34](https://github.com/Cray-HPE/metal-provision/assets/7772179/7370de5a-8d5f-4dd7-82b2-aee6063591d4)

Reply:
Re-ran this content after purging `/var/www/boot`, looks good.


```bash
redbull-management:~ # rm -rf /var/www/boot/*
redbull-management:~ # source /opt/cray/ansible/bin/activate
(ansible) redbull-management:~ # ansible-playbook /srv/cray/metal-provision/bootserver.yml
# omitted for brevity
(ansible) redbull-management:~ # cd /var/www/boot
(ansible) redbull-management:/var/www/boot # ll
total 4
lrwxrwxrwx 1 root    root   73 Sep  8 20:11 bootaa64.efi -> /usr/share/tftpboot-installation/SLE-15-SP5-aarch64/EFI/BOOT/bootaa64.efi
lrwxrwxrwx 1 root    root   71 Sep  8 20:11 bootx64.efi -> /usr/share/tftpboot-installation/SLE-15-SP5-x86_64/EFI/BOOT/bootx64.efi
-rw-r--r-- 1 dnsmasq tftp 1722 Sep  8 20:11 grub.cfg
lrwxrwxrwx 1 root    root   36 Sep  8 20:11 grub.efi -> /usr/share/grub2/x86_64-efi/grub.efi
(ansible) redbull-management:/var/www/boot # ll /usr/share/tftpboot-installation/SLE-15-SP5-aarch64/EFI/BOOT/bootaa64.efi
-rwxr-xr-x 1 root root 878800 May 18 11:49 /usr/share/tftpboot-installation/SLE-15-SP5-aarch64/EFI/BOOT/bootaa64.efi
(ansible) redbull-management:/var/www/boot # ll /usr/share/tftpboot-installation/SLE-15-SP5-x86_64/EFI/BOOT/bootx64.efi
-rwxr-xr-x 1 root root 953800 May 18 11:31 /usr/share/tftpboot-installation/SLE-15-SP5-x86_64/EFI/BOOT/bootx64.efi
(ansible) redbull-management:/var/www/boot # ll /usr/share/grub2/x86_64-efi/grub.efi
-rw-r--r-- 1 root root 1275904 Jul  6 10:20 /usr/share/grub2/x86_64-efi/grub.efi
```